app/assets/javascripts/helpers/format_input_helper.es6.jsx

```diff
@@ -0,0 +1,123 @@
+/**
```

Comment:
I think we should create a new file in components called helpers instead of putting this in the javascript/helpers folder (this is for js methods, instead of react components)